src/Collectors/MethodCall/MethodCallCollector.php

```diff
@@ -50,13 +50,10 @@ public function processNode(Node $node, Scope $scope): ?array
             return null;
         }
 
-        $classReflection = $scope->getClassReflection();
-        if (! $classReflection instanceof ClassReflection) {
```

Comment:
before this PR, this check made collection on code which is not within class-scope early exiting.